### ISSUE TITLE
pdf.js rename master to main branch

### ISSUE THREAD

**Issue by tiekom** [mozilla/pdf.js#14263]
Since some months, GitHub put out a guidance for changing master to main branches. [Here](https://github.com/github/renaming) you have the detailed overview of the new guidance's from the GitHub team.

 Could it be possible to implement this guidance's to the pdf.js repository?

**Comment by anantraghuvanshi** [comment]
Yes, it could be done but there are certain things which be considered as per the guidance. It says, 
Renaming a branch will:

-Re-target any open pull requests
-Update any draft releases based on the branch
-Move any branch protection rules that explicitly reference the old name
-Update the branch used to build GitHub Pages, if applicable
-Show a notice to repository contributors, maintainers, and admins on the repository homepage with instructions to update local copies of the repository
-Show a notice to contributors who git push to the old branch
-Redirect web requests for the old branch name to the new branch name
-Return a "Moved Permanently" response in API requests for the old branch name


We could use branch -m to move the branch locally instead of creating a new one and then we could push the new branch to GitHub. 
@tiekom
